test/libs/generate_api_v2_test.py

```diff
@@ -54,15 +54,15 @@ def test_build_summary_for_fips(
             negativeTests=nyc_latest["negative_tests"],
             hospitalBeds={
                 "capacity": nyc_latest["max_bed_count"],
-                "currentUsageCovid": None,
-                "currentUsageTotal": None,
+                "currentUsageCovid": nyc_latest["current_hospitalized"],
+                "currentUsageTotal": nyc_latest["hospital_beds_in_use_any"],
```

Comment:
These all get populated once we plumb in the HHS data so I had to update the test.